src/lang/af.js

```diff
@@ -1 +1,77 @@
-AlloyEditor.Strings = {"alignCenter":"Middel","alignJustify":"Eweredig","alignLeft":"Links","alignRight":"Regs","bold":"Vet","bulletedlist":"Ongenommerde lys","cancel":"Kanselleer","horizontalrule":"Horisontale lyn invoeg","italic":"Skuins","numberedlist":"Genommerde lys","quote":"Sitaatblok","removeformat":"Verwyder opmaak","strike":"Deurgestreep","subscript":"Onderskrif","superscript":"Bo-skrif","underline":"Onderstreep","formatted":"Opgemaak","h1":"Opskrif 1","h2":"Opskrif 2","normal":"Normaal","indent":"Vergroot inspring","outdent":"Verklein inspring","blockStyles":"Blok style","inlineStyles":"Inlyn style","objectStyles":"Objek style","styles":"Styl","cell":"Sel","cellDelete":"Verwyder sel","cellInsertAfter":"Voeg sel in na","cellInsertBefore":"Voeg sel in voor","cellMerge":"Voeg selle saam","cellMergeDown":"Voeg saam ondertoe","cellMergeRight":"Voeg saam na regs","cellSplitHorizontal":"Splits sel horisontaal","cellSplitVertical":"Splits sel vertikaal","column":"Kolom","columnDelete":"Verwyder kolom","columnInsertAfter":"Voeg kolom in na","columnInsertBefore":"Voeg kolom in voor","deleteTable":"Verwyder tabel","headers":"Opskrifte","headersBoth":"Beide    ","headersColumn":"Eerste kolom","headersNone":"Geen","headersRow":"Eerste ry","row":"Ry","rowDelete":"Verwyder ry","rowInsertAfter":"Voeg ry in na","rowInsertBefore":"Voeg ry in voor","add":"Add","ariaUpdateNoToolbar":"No toolbars are available","ariaUpdateOneToolbar":"{toolbars} toolbar is available. Press ALT+F10 to focus.","ariaUpdateManyToolbars":"{toolbars} toolbars are available. Press ALT+F10 to focus.","camera":"Insert Image from Camera","cameraDisabled":"The browser does not support this action, or it is available on https only (Chrome).","cite":"Cite","clearInput":"Clear Input Field","code":"Code","columns":"Cols","confirm":"Confirm","deleteEmbed":"Delete embed","editLink":"Type or paste link here","image":"Insert Image","link":"Link","linkTargetBlank":"_blank (new tab)","linkTargetDefault":"default","linkTargetParent":"_parent","linkTargetSelf":"_self (same tab)","linkTargetTop":"_top","removeLink":"Remove link","rows":"Rows","table":"Insert Table"};
\ No newline at end of file
+AlloyEditor.Strings = {
```

Comment:
All these lang files look auto-generated, so perhaps they should be excluded (ie. with a `@noformat` annotation in the header).

Reply:
Yes you're totally right, these are generated.